### PR TITLE
Map

### PR DESCRIPTION
**Pull request by cassiemak**
- Google Map - markers / info windows - done
- Info Panel & Filter box - done
- Augular Slider in Filter box - done
- SCSS color files - done

### CODE REVIEW

**Comment by h8rry** [comment]
You need to gitignore `log/development.log` and `tmp`. The file changes look scary.

**Comment by cassiemak** [comment]
i thought we did???

# Ignore all logfiles and tempfiles.

/log/*.log
/tmp

**Comment by h8rry** [comment]
You need to remove the files manually afterwards.

**Comment by cassiemak** [comment]
I dont get it.
so what do I need to put in the gitignore file to avoid this?

**Comment by h8rry** [comment]
You just have to remove the files now and push it back to github again

**Comment by cassiemak** [comment]
i thought i did that already when i rebased...

**Comment by h8rry** [comment]
I don't know. It still shows up: https://github.com/cchchoi1986/hikehk/pull/23/files

**Comment by cassiemak** [comment]
This is so weird...........

**Comment by h8rry** [comment]
Is this fixed yet?

**Comment by cassiemak** [comment]
Don't know how to fix....

**Comment by h8rry** [comment]
Untrack a file with `git rm --cached filename`

http://source.kohlerville.com/2009/02/untrack-files-in-git/

**Comment by h8rry** [comment]
I have removed them already

File: app/assets/javascripts/static_pages.js [hunk 1]
```diff
@@ -1 +1,219 @@
 
+// $(document).ready(function(){
```
Comment (h8rry) [inline app/assets/javascripts/static_pages.js]: We should remove this file

Comment (cassiemak) [inline app/assets/javascripts/static_pages.js]: Pls merge after you did your thing

Comment (h8rry) [inline app/assets/javascripts/static_pages.js]: What do you mean?

Comment (cassiemak) [inline app/assets/javascripts/static_pages.js]: You said you fixed it? So merge after you fixed it. Then I will pull
request

Comment (h8rry) [inline app/assets/javascripts/static_pages.js]: I have only fixed the gitignore issues. Please fix other issues.

Cassie Mak notifications@github.com于2015年2月17日星期二写道：

> In app/assets/javascripts/static_pages.js
> https://github.com/cchchoi1986/hikehk/pull/23#discussion_r24822273:
> 
> > @@ -1 +1,219 @@
> > 
> > +// $(document).ready(function(){
> 
> You said you fixed it? So merge after you fixed it. Then I will pull
> request On 17 Feb 2015 23:14, "Harry Chen" <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
> … <#14b9820745c77980_>
>  In app/assets/javascripts/static_pages.js <
> https://github.com/cchchoi1986/hikehk/pull/23#discussion_r24821810>: > @@
> -1 +1,219 @@ > > +// $(document).ready(function(){ What do you mean? —
> Reply to this email directly or view it on GitHub <
> https://github.com/cchchoi1986/hikehk/pull/23/files#r24821810>.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/cchchoi1986/hikehk/pull/23/files#r24822273.

## 

_Harry Chen_
Website http://www.harrychen.net | LinkedIn
http://ca.linkedin.com/in/zengjinchen

Comment (cassiemak) [inline app/assets/javascripts/static_pages.js]: What is the other issue

Btw clement doesn't get notifications

Comment (h8rry) [inline app/assets/javascripts/static_pages.js]: Please go to github and read through the comments @cchchoi1986 @cassiemak

**Pull request merged**